Removes desktop specific if statements and gtk-update-icon-cache command since there's already a trigger for that

diff --git a/install.sh b/install.sh
index dfc45a99..1868494a 100755
--- a/install.sh
+++ b/install.sh
@@ -52,12 +52,8 @@ install() {
   cp -r ${SRC_DIR}/{COPYING,AUTHORS}                                                   ${THEME_DIR}
   cp -r ${SRC_DIR}/src/index.theme                                                     ${THEME_DIR}
 
-  if [[ $DESKTOP_SESSION == '/usr/share/xsessions/plasma' && ${color} == '' ]]; then
-    sed -i "s/Adwaita/breeze/g" ${THEME_DIR}/index.theme
-  fi
-
-  if [[ $DESKTOP_SESSION == '/usr/share/xsessions/plasma' && ${color} == '-dark' ]]; then
-    sed -i "s/Adwaita/breeze-dark/g" ${THEME_DIR}/index.theme
+  if [[ ${color} == '-dark' ]]; then
+    sed -i "s/breeze/breeze-dark/g" ${THEME_DIR}/index.theme
   fi
 
   cd ${THEME_DIR}
@@ -80,7 +76,7 @@ install() {
   fi
 
 
-  if [[ ${color} == '' && $DESKTOP_SESSION == '/usr/share/xsessions/budgie-desktop' ]]; then
+  if [[ ${color} == '' ]]; then
     cp -r ${SRC_DIR}/src/status/symbolic-budgie/*.svg                                  ${THEME_DIR}/status/symbolic
   fi
 
@@ -134,9 +130,6 @@ install() {
   ln -sf mimes mimes@2x
   ln -sf places places@2x
   ln -sf status status@2x
-
-  cd ${dest}
-  gtk-update-icon-cache ${name}${theme}${color}
 }
 
 while [[ $# -gt 0 ]]; do
diff --git a/src/index.theme b/src/index.theme
index f170745c..2087035c 100644
--- a/src/index.theme
+++ b/src/index.theme
@@ -1,7 +1,7 @@
 [Icon Theme]
 Name=Reversal
 Comment=MacOSX style icon theme for linux.
-Inherits=Numix-Circle-Light,Adwaita,hicolor
+Inherits=Numix-Circle-Light,breeze,Adwaita,hicolor
 Example=folder
 
 # KDE/Plasma Stuff
